target/linux/ramips/base-files/etc/board.d/02_network

```diff
@@ -511,6 +512,10 @@ ramips_setup_macs()
 	sk-wb8)
 		wan_mac=$(mtd_get_mac_binary factory 57350)
 		;;
+	f9k1109)
```

Comment:
@kapieyow Please use `belkin,f9k1109` as set in DTS compatible and if possible try to keep alphabetical ordering.